src/lib/utils.ts

```diff
@@ -1,4 +1,15 @@
+import type { AssetId } from '@shapeshiftoss/caip'
 export const getFeatureFlag = (flagName: string): boolean => {
   const envVarName = `VITE_FEATURE_${flagName.toUpperCase()}`
   return import.meta.env[envVarName] === 'true'
 }
+export const middleEllipsis = (value: string): string =>
+  value.length >= 12 ? `${value.slice(0, 4)}...${value.slice(-4)}` : value
+export const isEthAddress = (address: string): boolean => /^0x[0-9A-Fa-f]{40}$/.test(address)
```

Comment:
Let's use this guy instead instead of a RegEx https://viem.sh/docs/utilities/isAddress.html